src/Microhard/MicrohardManager.cc

```diff
@@ -25,23 +25,36 @@ static const char *kNET_MASK            = "NetMask";
 static const char *kCFG_USERNAME        = "ConfigUserName";
 static const char *kCFG_PASSWORD        = "ConfigPassword";
 static const char *kENC_KEY             = "EncryptionKey";
+static const char *kPAIR_CH             = "PairingChannel";
+static const char *kCONN_CH             = "ConnectingChannel";
 
 //-----------------------------------------------------------------------------
 MicrohardManager::MicrohardManager(QGCApplication* app, QGCToolbox* toolbox)
     : QGCTool(app, toolbox)
 {
+    for (int i = 1; i < 82; i++) {
```

Comment:
How about having the hard-coded values (nr. of channels and frequency offset ) as constants and a comment on where are they sourced from?

Reply:
Yes - I defined and commented the constants.